src/app/qgisapp.cpp

```diff
@@ -6507,6 +6508,55 @@ void QgisApp::newMeshLayer()
   dialog.exec();
 }
 
+void QgisApp::newGpxLayer()
+{
+  QgsSettings settings;
+  const QString dir = settings.value( QStringLiteral( "Plugin-GPS/gpxdirectory" ), QDir::homePath() ).toString();
```

Comment:
Hm, can we eventually replace that settings key string? We could fallback on this one if a cleaner key returns an empty value.

Reply:
Yep, I'll do when I remove the plugin for good